### PR TITLE
Added 50-10% subphase to Cerus

### PR DESCRIPTION
**Pull request by Linkaaaaa**

### CODE REVIEW

File: GW2EIEvtcParser/EncounterLogic/Strikes/SotO/TempleOfFebe.cs [hunk 1]
```diff
@@ -208,6 +209,16 @@ internal override List<PhaseData> GetPhases(ParsedEvtcLog log, bool requirePhase
                 var phase = new PhaseData(enragedSmash.Time, log.FightData.FightEnd, "Enraged Smash");
                 phase.AddTarget(mainTarget);
                 phases.Add(phase);
+                // Sub Phase for 50-10%
+                for (int i = 1; i <= invulnPhases.Count; i++)
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EncounterLogic/Strikes/SotO/TempleOfFebe.cs:213]: var phase3 = invulPhases.LastOrDefault(x => x.InInterval(enragedSmash.Time));
null check on phase3 then use its start.
So no failed breakbar phases?

Comment (Linkaaaaa) [reply GW2EIEvtcParser/EncounterLogic/Strikes/SotO/TempleOfFebe.cs:213]: No just 50-10 until the breakbar finally breaks

**Comment by EliphasNUIT** [comment]
So we are 100% certain about not having breakbar failed phases?

**Comment by Linkaaaaa** [comment]
Last time you said you didn't want indivisual phases for failed breakbars, so don't think so? Or how do you want to handle those

**Comment by EliphasNUIT** [comment]
Not personaly interested in seeing it handled, I was asking about it as it was a topic of discussion.
Thanks!

**Pull request merged**